### PR TITLE
[BAEL-14274] - Fixed article code for https://www.baeldung.com/jackso…

### PR DESCRIPTION
**Pull request by amit2103**
…n-annotations

### CODE REVIEW

File: jackson-simple/src/test/java/com/baeldung/jackson/date/Event.java [hunk 1]
```diff
@@ -4,17 +4,17 @@
 
 import com.fasterxml.jackson.annotation.JsonFormat;
 
-public class EventWithFormat {
+public class Event {
```
Comment (lor6) [inline jackson-simple/src/test/java/com/baeldung/jackson/date/Event.java]: I think the EventWithFormat is for a different article; instead we should have the EventWithSerializer in the article

File: jackson-simple/src/test/java/com/baeldung/jackson/date/Event.java [hunk 2]
```diff
@@ -4,17 +4,17 @@
...
     public String name;
 
     @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy hh:mm:ss")
     public Date eventDate;
 
-    public EventWithFormat() {
+    public Event() {
```
Comment (lor6) [inline jackson-simple/src/test/java/com/baeldung/jackson/date/Event.java]: please check the updated article: https://www.baeldung.com/jackson-annotations 
the Event classes should not be changed here

Comment (amit2103) [reply jackson-simple/src/test/java/com/baeldung/jackson/date/Event.java]: Done, reverted the rename

File: jackson-simple/src/test/java/com/baeldung/jackson/date/Event.java (hunk 1 above)
Comment (amit2103) [reply jackson-simple/src/test/java/com/baeldung/jackson/date/Event.java]: Done, reverted the rename

**Pull request merged**